app/controllers/manage_cmr/permissions_controller.rb

```diff
@@ -197,18 +197,22 @@ def set_collection_permission
   # and hydrates the `group` key with the group details and `is_hidden`
   # with a boolean representing the users ability to see this group
   def hydrate_groups(permission)
+    group_permissions = permission.fetch('group_permissions', [])
+
+    permission['group_permissions'] = group_permissions
     permission.fetch('group_permissions', []).each do |group_permission|
       next unless group_permission.key?('group_id')
 
-      if group_permission['group_id'] =~ /(-CMR)$/ && !policy(:system_group).read?
-        # If this user does not have access to view this group mark it for hiding
-        group_permission['is_hidden'] = true
+      group_response = cmr_client.get_edl_group(group_permission['group_id'])
 
-        next
+      if group_response.success?
+        provider_id = group_response.body["tag"]
+        # If this user does not have access to view this group mark it for hiding
+        if provider_id == "CMR" && !policy(:system_group).read?
```

Comment:
Yes, the provider_id is considered the tag and this piece of functionality wants to hide the group.   The old way used to look at the concept id, and if it had CMR in it, it would determine that way if it should be hidden.   EDL has that information in the tags, so that is why it is looking in there.